### PR TITLE
Changes the way we evaluate policies to using entrypoints

### LINKED ISSUE #143: [FEAT] Allow setting `entrypoint` in `opaClient` rather than `package`
It wasn't clear from the docs what the name of the rule which was to be evaluated should be, as only the `package` was requested in configuration. The name shouldn't matter however, and I think it would be preferable if the user could set an `entrypoint` instead of a package. Entrypoints would be either a package or a package/rule combination and using any level of nesting, like `policy`, `policy/backstage/permissions` or `policy/backstage/permissions/decision`.

### PR DESCRIPTION
**Pull request by Parsifal-M**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Changes the way we evaluate policies, we move to using entrypoints which is discussed in #143 

## Related Tickets & Documents

- Related Issue #143 
- Closes #143 

## Instructions, Screenshots, Recordings

N/A

## Added/updated tests?

_We encourage you to keep the code coverage percentage at 80% and above._

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

### CODE REVIEW

File: plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts [hunk 1]
```diff
@@ -32,22 +32,22 @@ export class OpaClient {
   /**
    * Evaluates a policy against a given input.
    * @param input - The input to evaluate the policy against.
-   * @param opaPackage - The OPA package to use. You can optionally provide the package here, otherwise it will be taken from the app-config.
+   * @param opaEntryPoint - The OPA package to use. You can optionally provide the package here, otherwise it will be taken from the app-config.
```
Comment (Parsifal-M) [inline plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts:35]: Forgot to update this, will do that.

**Comment by Parsifal-M** [comment]
@anderseknert can I get your opinion on this? 😅

File: plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts [hunk 2]
```diff
@@ -32,22 +32,22 @@ export class OpaClient {
...
 
     if (!opaBaseUrl) {
       this.logger.error('The OPA URL is not set in the app-config!');
       throw new Error('The OPA URL is not set in the app-config!');
     }
 
-    if (!setOpaPackage) {
+    if (!setEntryPoint) {
       this.logger.error(
         'The OPA package is not set in the evaluatePolicy method or in the app-config!',
```
Comment (anderseknert) [inline plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts:52]: Should say entrypoint here rather than package I guess

**Review by anderseknert — APPROVED** [review]
LGTM! Just an error message that might need to be updated.

File: plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts (hunk 2 above)
Comment (Parsifal-M) [reply plugins/permission-backend-module-opa-wrapper/src/opa-client/opaClient.ts:52]: Ah yes! ty!

**Pull request merged**